portfolio/src/main/webapp/style.css

```
@@ -7,3 +7,108 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+}
+body {
+    background-image: url('https://img.freepik.com/free-photo/white-concrete-wall_53876-92803.jpg');
+}
+h1 {text-align: center;}
+*{
+padding:0;
+margin:0;
```

[4.2.4](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space after a property name’s colon.
```suggestion
padding: 0;
margin: 0;
```